### DISCUSSION TITLE
Please consider change tagging convention

### DISCUSSION THREAD

**Discussion by kloczek** [HDFGroup/hdf4#552] [Proposed Change]
Is it possible on next release change tagging convention?

Currently is used `hdf-<version_with_undescores_and-dash>`.
`-` is character which is forbidden to be used in version strings with most of the packaging software.
To make packaging easier it would be good to change that to just `<version.with.dots>` or `v<version.with.dots>` (in autogenerated from git tag tar ball this leading v is skipped) like it is in most of the other cases.

Possible to do that?🤔

**Comment by derobins** [comment]
We'll need to discuss this and also consider this for HDF5.

**Comment by derobins** [comment]
Can you provide a link to documentation that describes what the rules are for names?
